src/main/java/com/uid2/shared/auth/OperatorKey.java

```diff
@@ -105,6 +102,13 @@ public static OperatorKey valueOf(JsonObject json) {
                 );
     }
 
+    private Set<Role> reorderAndAddDefaultRole(Set<Role> roles) {
```

Comment:
Is order important?

Reply:
i notice that when some other unrelated OperatorKey entries or fields are updated, the Roles ordering changed  such as this diff:

![image](https://user-images.githubusercontent.com/2499852/212270617-9fb2f2ae-17c4-4fbe-954c-9e66e88b8d52.png)

This could potentially cause us to think a diff has been created and therefore we need to write private site data again in UID-575. just FYI @aulme

Reply:
Makes sense! Thanks